Cache/Redis.php

```diff
@@ -4,6 +4,7 @@
 
 use Handlebars\Cache;
 use Predis\Client as Predis;
+use Redis;
```

Comment:
You have to remove that, since we cannot rely on the user having the `phpredis` extension installed.